client/subscription-edit-page/index.tsx

```diff
@@ -0,0 +1,280 @@
+/* eslint-disable prettier/prettier */
+/* global jQuery */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { createRoot } from 'react-dom/client';
+import { __ } from '@wordpress/i18n';
+
+// TypeScript declaration for jQuery
+declare const jQuery: (
+	selector: any
+) => {
+	on: ( event: string, handler: () => void ) => void;
+	off: ( event: string, handler: () => void ) => void;
+};
+
+/**
+ * Internal dependencies
+ */
+import type {
+	PaymentMethodSelectProps,
+	WCPayPMSelectorData,
+	FetchUserTokensResponse,
+} from './types';
+import UserTokenCache from './user-token-cache';
+
+/**
+ * Add a listener to the customer select.
+ *
+ * This could be a shorter method, but because the customer select
+ * element uses select2, it does not emit the typical `change` event.
+ *
+ * @param {(userId: number) => void} callback The callback to call when the customer is changed.
+ * @return {() => void} The cleanup function.
+ */
+const addCustomerSelectListener = (
+	callback: ( userId: number ) => void
+): ( () => void ) => {
+	const customerUserSelect = document.getElementById(
+		'customer_user'
+	) as HTMLSelectElement | null;
+
+	if ( ! customerUserSelect ) {
+		return (): void => {
+			// No-op cleanup function when element is not found
+		};
+	}
+
+	// Wrap in an internal callback to load the select's value.
+	const internalCallback = () =>
+		callback( parseInt( customerUserSelect.value, 10 ) || 0 );
+
+	// Add the listner with the right technique, as select2 does not emit <select> events.
```

Comment:
```suggestion
	// Add the listener with the right technique, as select2 does not emit <select> events.
```

Reply:
Typo fixed in https://github.com/Automattic/woocommerce-payments/pull/11143/commits/af31e44dc5a3555cd4b142860c6633520c9e7579